terraform/stacks/dns/validators.tf

```diff
@@ -30,7 +30,15 @@ resource "aws_route53_record" "cloud_gov_zendesk_support_cname" {
   name    = "support.cloud.gov."
   type    = "CNAME"
   ttl     = 60
-  records = ["cloud-gov.zendesk.com."]
+  records = ["cloud-gov-new.zendesk.com."]
```

Comment:
Just to confirm, we don't need to preserve the old record?

Reply:
I believe they are shutting down the old subdomain `cloud-gov.zendesk.com` along with the old instance it is associated to as apart of the Zendesk migration.